dsmr_parser/telegram_specifications.py

```diff
@@ -430,11 +430,6 @@
             'value_parser': CosemParser(ValueParser(str)),
             'value_name': 'TEXT_MESSAGE'
         },
-        {
-            'obis_reference': obis.DEVICE_TYPE,
-            'value_parser': CosemParser(ValueParser(int)),
-            'value_name': 'DEVICE_TYPE'
-        },
```

Comment:
May be keep de generic one?

Reply:
Will cause same issue as above and is to generic as it will match an mbus device type.